services/weightscale.md

```diff
@@ -0,0 +1,43 @@
+# Weight Scale
+
+    identifier: 0x1f4d5040
+    extends: _sensor
+
+A weight measuring sensor.
+
+## Registers
+
+    ro weight: u16.16 g @ reading
```

Comment:
This is up to 65kg, which is not good for human scales.

I think we should use u22.10 - up to 4 tons and with 0.001g precision